Assets/MixedRealityToolkit/_Core/Definitions/Utilities/Handedness.cs

```diff
@@ -30,13 +30,13 @@ public enum Handedness : byte
         /// </summary>
         Both = Left | Right,
         /// <summary>
-        /// Reserved, for systems that provide alternate hand state.
-        /// </summary>
-        Other = 4,
-        /// <summary>
         /// Global catchall, used to map actions to any controller (provided the controller supports it)
         /// </summary>
         /// <remarks>Note, by default the specific hand actions will override settings mapped as both</remarks>
-        Any = None | Other | Left | Right,
+        Any = None | Left | Right,
+        /// <summary>
+        /// Reserved, for systems that provide alternate hand state.
+        /// </summary>
+        Other = 4,
```

Comment:
Still don't think we need an Other. Both also seems superfluous until we encounter a platform that has a controller that requires both hands.

Let's just go with None, Left, Right and Any

Reply:
BTW, Or-ing in a value of 0 still makes Both == Any

Reply:
@SimonDarksideJ ?